dart/gui/osg/CMakeLists.txt

```diff
@@ -8,11 +8,10 @@ if(DART_BUILD_GUI_OSG)
 
   find_package(OpenSceneGraph 3.0 QUIET
     COMPONENTS osg osgViewer osgManipulator osgGA osgDB)
-  if(OPENSCENEGRAPH_FOUND)
+  if(OPENSCENEGRAPH_FOUND AND OSG_FOUND)
```

Comment:
Are all these variables are set to true when OSG is found? It seems `OPENSCENEGRAPH_FOUND` is defined when [`FindOpenSceneGraph.cmake`](https://github.com/Kitware/CMake/blob/master/Modules/FindOpenSceneGraph.cmake) is used while `OSG_FOUND` is defined when [`FindOSG.cmake`](https://github.com/openscenegraph/OpenSceneGraph/blob/1bdd2aceaba5bbc91fe174d943f63b0de754ec76/CMakeModules/FindOSG.cmake) is used. Shouldn't this to be `OPENSCENEGRAPH_FOUND OR OSG_FOUND`? Also, if `FindOSG.cmake` is used, it looks like we need to use `OSG_LIBRARY` instead of `OPENSCENEGRAPH_LIBRARIES`. Am I missing something here?